Sources/StreamChatSwiftUI/ChatChannelList/MoreChannelActionsView.swift

```diff
@@ -49,18 +51,31 @@ public struct MoreChannelActionsView: View {
                         Divider()
                             .padding(.horizontal, -16)
                         
-                        Button {
-                            if action.confirmationPopup != nil {
-                                viewModel.alertAction = action
-                            } else {
-                                action.action()
+                        if let destination = action.navigationDestination {
+                            Button {
+                                presentedView = destination
```

Comment:
Maybe we can make these properties dependent? For example, on the didSet of `presentedView` we could change the `isPresented` bool value. With the current approach, we have to always set the two values together.